sdccc/src/test/java/com/draeger/medical/sdccc/messages/TestMessageStorage.java

```diff
@@ -1192,6 +1193,8 @@ public void testGetInboundMessagesByBodyTypeAndSequenceId(@TempDir final File di
                 message.write(messageContent2.getBytes(StandardCharsets.UTF_8));
             }
 
+            Thread.sleep(5);
```

Comment:
Why was this change introduced?

Reply:
When executing this test via Maven, it sometimes happened that the two messages had the same timestamp and where hence retrieved from the DB in the wrong order. I hence introduced this sleep of 5ms to ensure differing timestamps and hence stabilize the test.

Reply:
Done.